Explorer/Assets/DCL/AvatarRendering/Emotes/Systems/Play/EmotePlayer.cs

```diff
@@ -102,20 +110,30 @@ public bool Play(GameObject mainAsset, AudioClip? audioAsset, bool isLooping, bo
         }
 
         private static bool IsValid(GameObject mainAsset) =>
-            mainAsset.GetComponent<Animator>();
+            mainAsset.GetComponent<Animator>() || mainAsset.GetComponent<Animation>();
```

Comment:
As a performance improvement, could you only check `Animation` component if `localSceneDevelopment:true`?

Reply:
tackled at https://github.com/decentraland/unity-explorer/pull/3390/commits/90f6dbe9f72c42ed456fffe663c410cf21e0daa9